core/src/main/kotlin/org/evomaster/core/problem/rest/UsedObjs.kt

```diff
@@ -0,0 +1,91 @@
+package org.evomaster.core.problem.rest
+
+import org.evomaster.core.search.gene.*
+
+class UsedObjs {
```

Comment:
abbreviated words are fine for local variables. but, for classes, I would prefer something more explicit like UsedObjects